src/components/Footer.tsx

```diff
@@ -1,82 +1,161 @@
-import { FaDev, FaDiscord, FaGithub, FaTwitter, FaYoutube } from "react-icons/fa";
+import { FaDev, FaDiscord } from "react-icons/fa";
+import OpenSaucedLogo from "../assets/openSauced-icon.png";
+
+// icons
```

Comment:
this could go before the static open sauced logo however sorting imports is not enforced on purpose :D